src/views/forms/FormNamespaceRegistrationTransaction/FormNamespaceRegistrationTransaction.vue

```diff
@@ -1,73 +1,61 @@
-                  {{ $t('option_root_namespace') }}
-                </Option>
-                <Option
-                  v-if="ownedNamespaces.length"
-                  :value="typeSubNamespace"
-                >
-                  {{ $t('option_sub_namespace') }}
-                </Option>
-              </Select>
+            <div class="inputs-container">
+              <div :value="typeRootNamespace" v-if="$route.path.substring(1)==='createNamespace'" class="display-value">
+                {{ $t('option_root_namespace') }}
+              </div>
+              <div v-if="$route.path.substring(1)==='createSubNamespace'&&ownedNamespaces.length"
+                :value="typeSubNamespace" class="display-value">
+                {{ $t('option_sub_namespace') }}
+              </div>
             </div>
           </template>
         </FormRow>
 
-        <NamespaceSelector
-          v-if="formItems.registrationType === typeSubNamespace && ownedNamespaces.length"
-          v-model="formItems.parentNamespaceName"
-          label="form_label_parent_namespace"
-          :namespaces="fertileNamespaces"
-        />
-
-        <NamespaceNameInput
-          v-model="formItems.newNamespaceName"
-          :namespace-registration-type="formItems.registrationType"
-        />
-
-        <DurationInput
-          v-if="formItems.registrationType === typeRootNamespace"
-          v-model="formItems.duration"
-          target-asset="namespace"
-        />
+       <!--  <NamespaceSelector v-if="formItems.registrationType === typeSubNamespace && ownedNamespaces.length"
+          v-model="formItems.parentNamespaceName" label="form_label_parent_namespace" :namespaces="fertileNamespaces" />  -->
+          <NamespaceSelector v-if="formItems.registrationType === typeSubNamespace && ownedNamespaces.length"
+          :value="formItems.parentNamespaceName" @input="setParentNamespaceName" label="form_label_parent_namespace" :namespaces="fertileNamespaces" /> 
+        <NamespaceNameInput v-model="formItems.newNamespaceName" :is-need-auto-focus="false"
+          :namespace-registration-type="formItems.registrationType" />
+        <FormRow v-if="formItems.registrationType === typeSubNamespace">
+          <template v-slot:label>
+            {{$t('current_validity')}}:
+          </template>
+          <template v-slot:inputs>
+            <div class="inputs-container">
+              <div class="display-value">
+                {{relativeTimetoParent}}
+              </div>
+            </div>
+          </template>
+        </FormRow>
 
-        <MaxFeeAndSubmit
-          v-model="formItems.maxFee"
-          @button-clicked="handleSubmit(onSubmit)"
-        />
+        <DurationInput v-if="formItems.registrationType === typeRootNamespace" v-model="formItems.duration"
+          target-asset="namespace" :show-relative-time="true"/>
+        
+        <MaxFeeAndSubmit v-model="formItems.maxFee" @button-clicked="handleSubmit(onSubmit)" />
```

Comment:
please reapply indentation (spaces). It is hard to tell _what changed_, like this.